### ISSUE TITLE
Keyword Open Accessibility Test Browser doesn't work with newer versions of Firefox

### ISSUE THREAD

**Issue by idgserpro** [datakurre/robotframework-selenium2accessibility#2]
Keyword `Open Accessibility Test Browser` doesn't work with newer versions of Firefox. Using Firefox 72, I get the error:
```
AssertionError: Parent suite setup failed:
WebDriverException: Message: [Exception... "The URI is malformed"  nsresult: "0x804b000a (NS_ERROR_MALFORMED_URI)"  location: "JS frame :: chrome://marionette/content/listener.js :: get/< :: line 1138"  data: no]
```
